app/assets/javascripts/stradivari/helpers.js

```diff
@@ -40,6 +40,27 @@ window._TABLE_ = (function() {
         concat(params)
       );
 
+    },
+
+    // example
+    // in: http://mysite.com?stradi_tabs[]=bar&stradi_tabs[]=baz&foo=bar
+    // out: {stradi_tabs: ['bar', 'baz'], foo: "bar"}
+    parseURLParameters: function(uri) {
+      var re = /([^&=]+)=?([^&]*)/g;
+      var decodeRE = /\+/g;  // Regex for replacing addition symbol with a space
+      var decode = function (str) {return decodeURIComponent( str.replace(decodeRE, " ") );};
+      var query = uri.split('?')[1]
```

Comment:
mixed feelings, do we really want to depend on the burden in taking that in the asset pipeline, or depending on a specific vendored version, or ... 

we have simple requirements, as such the simple code we have that it boils down to `example.com?foo=bar&bar=baz".split('?')[1].split('&')` should be enough... hairy regexen maybe not :-)

wdyt?